### PR TITLE
[INTEL MKL] Update default TF_BUILD_VERSION to r1.11 and add packages

### PR DESCRIPTION
**Pull request by congxu-ml**
Signed-off-by: Cong Xu <cong.xu@intel.com>

### CODE REVIEW

File: tensorflow/tools/docker/Dockerfile.devel-mkl-horovod [hunk 1]
```diff
@@ -29,7 +29,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
         openjdk-8-jdk \
         openjdk-8-jre-headless \
         wget \
-        numactl \
+        libnuma-dev \
```
Comment (gunan) [inline tensorflow/tools/docker/Dockerfile.devel-mkl-horovod]: FYI, we cannot distribute numa ourselves, but I think just having a dockerfile with reference to it is fine.
@martinwicke to confirm.

Comment (martinwicke) [reply tensorflow/tools/docker/Dockerfile.devel-mkl-horovod]: Yes, it's fine.

Comment (congxu-ml) [reply tensorflow/tools/docker/Dockerfile.devel-mkl-horovod]: Yes, we are not distributing binary, this is just a reference in Dockerfile. Thanks!

**Review by gunan — APPROVED** [review]

**Comment by gunan** [comment]
@yifeif any idea why we see the copybara error?

**Pull request merged**